opensrp-core/src/main/java/org/smartregister/task/SaveTeamLocationsTask.java

```diff
@@ -1,17 +1,18 @@
 package org.smartregister.task;
 
-import android.os.AsyncTask;
-
 import org.smartregister.location.helper.LocationHelper;
 
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
 /**
  * Created by ndegwamartin on 26/06/2018.
  */
 
-public class SaveTeamLocationsTask extends AsyncTask<Void, Void, Void> {
-    @Override
-    protected Void doInBackground(Void... params) {
-        LocationHelper.getInstance().locationIdsFromHierarchy();
-        return null;
+public class SaveTeamLocationsTask {
```

Comment:
Yeah that does. We expect some stuff to break with the major version release. I guess we could document the specific change for migration. For this particular case it should be refactored since Android deprecated AsyncTasks.

Reply:
Can we have a draft release note as a comment on the PR so that we can keep track of breaking changes?

This will help other guys who might need to migrate codebases so that they can sequentially fix breaking changes